File: src/main/java/no/nav/domain/pensjon/regler/Yrkesskadeopptjening.java
package no.nav.domain.pensjon.regler;

import java.io.Serializable;

public class Yrkesskadeopptjening implements Serializable {
    

    /*
     * Poeng etter antatt årlig inntekt.
     */
    private double paa;
    /*
     * Yrkesskadegrad.
     */
    private int yug;
    /*
     * Antatt Inntekt for yrkesskadeopptjeningen.
     */
    private double antattInntektYrke;

}

File: src/main/kotlin/no/nav/pensjon/regler/domain/Yrkesskadeopptjening.kt
package no.nav.pensjon.regler.domain

import java.io.Serializable

class Yrkesskadeopptjening : Serializable {
    /*
     * Poeng etter antatt årlig inntekt.
     */
    var paa = 0.0

    /*
     * Yrkesskadegrad.
     */
    var yug = 0

    /*
     * Antatt Inntekt for yrkesskadeopptjeningen.
     */
    var antattInntektYrke = 0.0
}